INSTALL.md

```diff
@@ -189,6 +189,8 @@ version of CP2K (ssmp or psmp), the FFTW3 threading library libfftw3_threads
 - Use `-D__CUDA_PROFILING` to turn on Nvidia Tools Extensions.
   It requires to link `-lnvToolsExt`.
 - Link to a blas/scalapack library that accelerates large DGEMMs (e.g. libsci_acc)
+- Use the `-D__COLLOCATE_GPU` flag to compile the gpu backend for collocate. It
```

Comment:
Do you want to have separate flags for collocate and integrate? Otherwise, I'd suggest to use `__GRID_CUDA`, which matches the `__PW_CUDA` that we already have. 

If you want to have separate flags for each backend then I'd suggest something along the lines of `_GRID_HYBRID_BACKEND`.